src/layouts/Media/Media.tsx

```diff
@@ -143,17 +181,27 @@ export const Media = (): JSX.Element => {
           </Box>
           <Skeleton
             w="100%"
-            h={isLoading ? "4.5rem" : "fit-content"}
-            isLoaded={!isLoading}
+            h={isListMediaFilesLoading ? "4.5rem" : "fit-content"}
+            isLoaded={!isListMediaFilesLoading}
           >
             <SimpleGrid columns={3} spacing="1.5rem" w="100%">
-              {mediasData?.filter(isMediaData).map(({ name, mediaUrl }) => {
+              {mediaFolderFiles?.files.map(({ name, mediaUrl }) => {
                 if (mediaType === "images") {
                   return <ImagePreviewCard name={name} mediaUrl={mediaUrl} />
                 }
                 return <FilePreviewCard name={name} />
               })}
             </SimpleGrid>
+            {mediaFolderFiles && mediaFolderFiles?.total > 0 && (
+              <Center mt="1rem">
+                <Pagination
+                  totalCount={mediaFolderFiles?.total || 0}
```

Comment:
nit: would the `0` ever happen? Seems like the conditional rendering would have eliminated the `0` here.

Reply:
done here to satisfy ts cos `totalCount` is given as a number. it was between this or using `!` to hint that it will always be present (but more prone to errors down the line)